packages/common/utils/src/scrollRestoration.en.md

```diff
@@ -1,4 +1,4 @@
-# scrollRestoration.set
```

Comment:
1. I think this is correct title because there is only `set` method in the `scrollRestoration` object, and I'm writing docs per method.
2. A reason why an unnecessary menu named `scrollRestoration` is created is there's no `@tossdocs-ignore` comment on the top of [scrollRestoration.ts](https://github.com/toss/slash/blob/d883a0b2aebdbc2ca39c67902cec754c63921dfe/packages/common/utils/src/scrollRestoration.ts#L1) file.

So could you keep the current title and add `@tossdocs-ignore` comment on the top of the file?

```typescript
/** @tossdocs-ignore */
```

Reply:
@HoseungJang 
Thank you for the detailed explanation. 🙇🏻‍♂️
The title has been restored, and `@tossdocs-ignore` has been added to the top of the `scrollRestoration.ts`.